### DISCUSSION TITLE
Houdoku Backup

### DISCUSSION THREAD

**Discussion by ScruffiX** [xgi/houdoku#182] [Feature Requests]
Hello,

is it possible to make Houdoku backups carry over chapter progress?
I seem to find it inconvenient that when switching devices read chapters don't carry over. I don't and can't keept track of what I've read for every entry and wouldn't want to check with scources where I do keep track of it everytime I load a backup.

**Comment by xgi** [comment]
Added with b1eb7f31dea9f89288942a337b2a81d65ae6ba33. Thanks!

**Comment by VoltKraken5555** [comment]
Please add an option to sync to and restore from cloud storage to track reading progress using multiple devices.
